src/composer.ts

```diff
@@ -615,7 +621,8 @@ export class AtomicTransactionComposer {
    *   in this group (ABIResult[]) and the SimulateResponse object.
    */
   async simulate(
-    client: Algodv2
+    client: Algodv2,
+    request?: SimulateRequest
```

Comment:
Can you add this param to the docs above with similar wording as the python SDK?

Reply:
see [pr comment response](https://github.com/algorand/js-algorand-sdk/pull/768/commits/fe95c4cbd087bffbfe7703092a28e0936a0e2186)